examples/tcp-close/client.rs

```diff
@@ -166,22 +146,19 @@ impl TcpClient {
         Ok(())
     }
 
-    /// Attempts to close several connections sequentially with the expectation
-    /// that the server will close sockets.
-    pub fn run_sequential_expecting_server_to_close_sockets(&mut self, nclients: usize) -> Result<()> {
-        for i in 0..nclients {
-            // Connect to the server and wait.
-            let qd: QDesc = self.issue_socket()?;
-            let qt: QToken = self.libos.connect(qd, self.remote)?;
-            let qr: demi_qresult_t = self.libos.wait(qt, Some(DEFAULT_TIMEOUT))?;
+    pub fn run_sequential_expecting_server_to_close_sockets(&mut self, num_clients: usize) -> Result<()> {
```

Comment:
Can you leave a comment somewhere (either beginning of the file or here) about what this test is trying to achieve? Similar to what I put at the top of tcp-push-pop. Because it's very unclear from the documentation what sequential vs concurrent means and why we want to have the server close vs the client close (and not test something else instead).

Reply:
Done, added comments at the top of both client and server programs.